src/library_pthread.js

```diff
@@ -706,7 +706,7 @@ var LibraryPThread = {
     var detached = 0;
     var schedPolicy = 0; /*SCHED_OTHER*/
     var schedPrio = 0;
-    if (attr) {
+    if (attr && attr != {{{ cDefine('__ATTRP_C11_THREAD') }}}) {
```

Comment:
Does this mean that `attr` means one thing when using C11 threads (it just has a defined value) but is something else otherwise (`makeGetValue` below)?

Reply:
Yeah the way musl implements C11 threads is:

```
#define __ATTRP_C11_THREAD ((void*)(uintptr_t)-1)
...

int ret = __pthread_create(thr, __ATTRP_C11_THREAD, (void *(*)(void *))func, arg);
```

Reply:
wow, thats.... very C-like.